mercury-operator/src/main/java/com/redhat/mercury/operator/controller/ServiceDomainController.java

```diff
@@ -74,14 +104,22 @@ public DeleteControl deleteResource(ServiceDomain sd, Context<ServiceDomain> con
             createOrUpdateDeployment(sd);
             createOrUpdateService(sd);
             if(ServiceDomainSpec.ExposeType.Http == sd.getSpec().getExpose()) {
```

Comment:
the expose should be an array of strings

Reply:
Done